applications/SeismicApplication/custom_conditions/fiber_beam_column_uniaxial_fiber.hpp

```diff
@@ -0,0 +1,283 @@
+//  KRATOS  ____       _               _
+//         / ___|  ___(_)___ _ __ ___ (_) ___
+//         \___ \ / _ \ / __| '_ ` _ \| |/ __|
+//          ___) |  __/ \__ \ | | | | | | (__
+//         |____/ \___|_|___/_| |_| |_|_|\___|
+//
+//  License:     BSD License
+//  license:     structural_mechanics_application/license.txt
+//
+//  Main authors: Mahmoud Zidan
+//    Co authors: Long Chen
+//
+
+#if !defined(KRATOS_FIBER_BEAM_COLUMN_UNIAXIAL_FIBER_H_INCLUDED )
+#define  KRATOS_FIBER_BEAM_COLUMN_UNIAXIAL_FIBER_H_INCLUDED
+
+// System includes
+
+
+// External includes
+
+
+// Project includes
+#include "includes/define.h"
+#include "includes/variables.h"
+#include "includes/element.h"
+
+namespace Kratos
+{
+
+///@}
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name  Kratos Classes
+///@{
+
+/**
+ * @class FiberBeamColumnUniaxialFiber
+ *
+ * @brief A 3D unaxial fiber for the beam-column element for reinforced concrete modeling
+ * @details it is stored in the section, in a container of fibers
+ *
+ * @author Mahmoud Zidan
+ */
+
+class KRATOS_API(SEISMIC_APPLICATION) FiberBeamColumnUniaxialFiber
```

Comment:
Idem to my previpous comments. header is .h, and if not a condition should be in a different folder